xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -348,3 +371,38 @@ void CAndroidUtils::OnSettingChanged(const std::shared_ptr<const CSetting>& sett
   if (settingId == CAndroidUtils::SETTING_LIMITGUI)
     CDisplaySettings::GetInstance().ClearCalibrations();
 }
+
+const std::vector<int> CAndroidUtils::GetDisplaySupportedHdrTypes()
```

Comment:
please never return plain const from functions.. either `const T&` or just `T` by value like in this case.

Reply:
I guess is just a confusion by https://github.com/xbmc/xbmc/pull/20422#discussion_r739639288

What @ksooo wanted is:
`std::vector<int> CAndroidUtils::GetDisplaySupportedHdrTypes() const`

Reply:
yes right :) sorry i didn't see this comment. this ^^ is what we want

Reply:
Okay, I got confused with the header being `const`.

Reply:
I know the review process is stressful sometimes... but we are like that ;-)

Reply:
No problem, this should be fixed now.